src/Nncase.Tests/Rules/Neutral/UnitTestCombineTranspose.cs

```diff
@@ -217,10 +219,10 @@ public void TestCombineTransposeConcatNegative(int[] inShape, int[][] perm, int
     public void TestCombineTransposeConstBinaryPositive(BinaryOp binaryOp, int[] lShape, int[] rShape, int[] perm, bool leftConst)
     {
         Expr lhs = leftConst ?
-          Const.FromValue(Random.Normal(DataTypes.Float32, 0, 1, 3, lShape).Evaluate()) :
+          lShape.Length == 0 ? 0.5f : Const.FromValue(Random.Normal(DataTypes.Float32, 0, 1, 3, lShape).Evaluate()) :
```

Comment:
Does this line should be reverted?

Reply:
I want to generate a scalar if shape is a empty array, shape.Length is not good?